src/js/form-builder.js

```diff
@@ -1007,24 +1007,38 @@ const FormBuilder = function(opts, element) {
   let appendNewField = function(values, isNew = true) {
     let type = values.type || 'text';
     let label = values.label || i18n[type] || i18n.label;
-    let delBtn = m('a', i18n.remove, {
+    let fieldButtons = [
+      m('a', i18n.remove, {
+        type: 'remove',
         id: 'del_' + data.lastID,
         className: 'del-button btn delete-confirm',
         title: i18n.removeMessage
+      }),
+      m('a', null, {
+        type: 'edit',
+        id: data.lastID + '-edit',
+        className: 'toggle-form btn icon-pencil',
+        title: i18n.hide
+      }),
+      m('a', null, {
+        type: 'copy',
+        id: data.lastID + '-copy',
+        className: 'copy-button btn icon-copy',
+        title: i18n.copyButtonTooltip
+      })
+    ];
+
+    let disabledFieldButtons = values.disabledFieldButtons;
```

Comment:
since this is coming from `values`, will generated fields have a `disabledFieldButtons` attribute?